build.sbt

```diff
@@ -1816,7 +1816,8 @@ lazy val flinkBaseComponentsTests = (project in flink("components/base-tests"))
   )
   .dependsOn(
     flinkComponentsTestkit  % Test,
-    flinkTableApiComponents % Test
+    flinkTableApiComponents % Test,
+    scenarioCompiler        % "test->test"
```

Comment:
Why was it needed?

Reply:
for reusing a component that operates on this newly added parameter